commons-mongo/src/main/scala/com/avsystem/commons/mongo/async/GenCodecCollection.scala

```diff
@@ -3,14 +3,15 @@ package mongo.async
 
 import com.avsystem.commons.mongo.core.GenCodecRegistry
 import com.avsystem.commons.serialization.GenCodec
-import com.github.ghik.silencer.silent
-import com.mongodb.async.client.{MongoCollection, MongoDatabase}
+import org.mongodb.scala.{MongoCollection, MongoDatabase}
 
 object GenCodecCollection {
```

Comment:
Does it differ from `com.avsystem.commons.mongo.scala.GenCodecCollection`? I guess we should remove this one now.

Reply:
Okay, deleted. I also moved to `MongoObservableExtensions` to `scala` package

Reply:
Let's call it MongoScalaObservableExtensions as they're no longer generic to Mongo (and the Mongo ones have different interface based on reactive streams)